Packs/Vectra_AI/Scripts/VectraDetectCloseDuplicateIncidents/VectraDetectCloseDuplicateIncidents.py

```diff
@@ -0,0 +1,69 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import traceback
+
+''' MAIN FUNCTION '''
+
+DEFAULT_PAGE_SIZE = 50
+DEFAULT_NOTE = 'Duplicate. Closed.'
+DEFAULT_INCIDENT_TYPE = 'Vectra Account, Vectra Host'
+
+
+def main():
+    try:
+        args = demisto.args()
+        page_size: int = arg_to_number(args.get('page_size', DEFAULT_PAGE_SIZE))  # type: ignore
```

Comment:
in general when writing this line `args.get('page_size', DEFAULT_PAGE_SIZE)` and the user send None as an argument (page_size=None), so the .get() func returns None and not the default value.

for example in line number 17, the user can send an empty list (incident_types=[]), so in this case, we don't have incident_types at all.

So consider it, you can change all those lines in this way ->

```
page_size: int = arg_to_number(args.get('page_size') or DEFAULT_PAGE_SIZE)
note = args.get('note') or DEFAULT_NOTE
incident_types = argToList(args.get('incident_types') or DEFAULT_INCIDENT_TYPE)
```

Reply:
Thanks @rshunim, for pointing out the edge scenario for arguments.

Now, we have used the `remove_space_from_args` and `remove_nulls_from_dictionary` to remove the keys of argument dictionary where value is set to None.

Also, we have updated the code for `incident_types` argument as per your suggested style.